src/main/resources/org/jenkinsci/plugins/aquadockerscannerbuildstep/AquaDockerScannerBuilder/global.jelly

```diff
@@ -4,25 +4,30 @@
     This Jelly script is used to produce the global configuration option.
   -->
   <f:section title="Aqua Security">
-  	<f:block>
-     <b>Choose aqua version:</b>
-   </f:block>
-   <f:radioBlock checked="${instance.isVersion('2.x')}" name="version" value="2.x" title="Version 2.x" inline="true">
-   </f:radioBlock>
-   <f:radioBlock checked="${instance.isVersion('3.x') or true}" name="version" value="3.x" title="Version 3.x and above" inline="true">
-   </f:radioBlock>
+ 
     <f:entry title="Aqua scanner image" field="aquaScannerImage">
       <f:textbox />
     </f:entry>
     <f:entry title="Aqua API URL" field="apiURL">
       <f:textbox />
     </f:entry>
-    <f:entry title="User" field="user">
-      <f:textbox />
-    </f:entry>
-    <f:entry title="Password" field="password">
-      <f:password />
-    </f:entry>
+    <f:radioBlock checked="${instance.isAuthType('uname') or true}" name="auth" value="uname" title="Username/Password for authentication">
```

Comment:
This operation will always return true

Reply:
This is for cases where nothing is selected, i.e. first time the plugin is installed. We had similar implementation for scanner version earlier:
![image](https://user-images.githubusercontent.com/33202950/150280779-5e2422fb-b12f-4e0c-bf6e-6e1405e65e41.png)